storage/innobase/srv/srv0start.cc

```diff
@@ -1231,6 +1231,20 @@ srv_open_tmp_tablespace(
 		it stays open until shutdown. */
 		if (fil_space_open(tmp_space->name())) {
 
+			if (srv_tmp_tablespace_encrypt) {
+
+				fil_space_t*		space;
+
+				space = fil_space_get(temp_space_id);
```

Comment:
Merge with declaration, declare as const

Reply:
cannot be const, but merged with declaration